oauth2_provider/management/commands/create_oauth2_client.py

```diff
@@ -73,13 +73,37 @@ def handle(self, *args, **options):
         self._clean_args(args)
         self._parse_options(options)
 
+        client_id = self.fields.get('client_id')
         trusted = self.fields.pop('trusted')
-        client = Client.objects.create(**self.fields)
-
+        trusted_client_exists = None
+
+        # Check if client ID is already in use. If so, fetch existing Client and update fields.
+        client_id_claimed = Client.objects.filter(client_id=client_id).exists()
```

Comment:
could this block simply be something like this? I think it would be easier to read than trying to keep track of new_client vs. old_client etc.

```python
client, was_created = Client.objects.get_or_create(client_id=client_id)
for key, value in self.fields.items():
    setattr(client, key, value)
client.save()

if trusted:
    TrustedClient.objects.get_or_create(client=client)
else:
    try:
        TrustedClient.objects.get(client=client).delete()
    except TrustedClient.DoesNotExist:
        pass
```